File: _Polynomial_Method_of_Defining_an_Ellipse.m

% Polynomial Method of Defining an Ellipse
a = 5;  % Semi-major axis
b = 3;  % Semi-minor axis

x = linspace(-a, a, 400);
y_upper = b * sqrt(1 - (x.^2 / a^2));
y_lower = -b * sqrt(1 - (x.^2 / a^2));

plot(x, y_upper, 'b', x, y_lower, 'b');
axis equal;
title('Polynomial Method of Defining an Ellipse');
xlabel('X-axis');
ylabel('Y-axis');
grid on;

File: _Polynomial_Method_of_Defining_an_Ellipse.py

import numpy as np
import matplotlib.pyplot as plt

# Ellipse parameters
a = 5   # Semi-major axis
b = 3   # Semi-minor axis

x = np.linspace(-a, a, 400)
y_upper = b * np.sqrt(1 - (x**2 / a**2))
y_lower = -b * np.sqrt(1 - (x**2 / a**2))

plt.plot(x, y_upper)
plt.plot(x, y_lower)
plt.gca().set_aspect('equal', adjustable='box')
plt.title("Polynomial Method of Defining an Ellipse")
plt.xlabel("X-axis")
plt.ylabel("Y-axis")
plt.grid(True)
plt.show()
